geomstats/geometry/poincare_ball.py

```diff
@@ -317,3 +317,268 @@ def inner_product_matrix(self, base_point=None):
         identity = gs.eye(self.dimension, self.dimension)
 
         return gs.einsum('i,jk->ijk', lambda_base, identity)
+
+
+NORMALIZATION_FACTOR_CST = gs.sqrt(gs.pi / 2)
+PI_2_3 = pow((2 * gs.pi), 2 / 3)
+SQRT_2 = gs.sqrt(2.)
+
+
+class GaussianDistribution():
+    """A class for Gaussian distributions."""
+
+    @staticmethod
+    def gaussian_pdf(data, means, variances, norm_func, metric):
```

Comment:
I assume this will be called as: ```GaussianDistribution.gaussian_pdf```, so maybe we don't really need to repeat `gaussian_`?